From 48020979d695864a392d9a3acde197c3d8592187 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 2 Jan 2019 10:26:16 +0100
Subject: [PATCH] udevadm: add a workaround for dracut

Also add a comment, so we don't fall into the same trap again.
---
 src/udev/udevadm.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/udev/udevadm.h b/src/udev/udevadm.h
index af0faff44..86b24af08 100644
--- a/src/udev/udevadm.h
+++ b/src/udev/udevadm.h
@@ -4,6 +4,7 @@
 #include <stdio.h>
 
 #include "build.h"
+#include "macro.h"
 
 int info_main(int argc, char *argv[], void *userdata);
 int trigger_main(int argc, char *argv[], void *userdata);
@@ -15,6 +16,7 @@ int test_main(int argc, char *argv[], void *userdata);
 int builtin_main(int argc, char *argv[], void *userdata);
 
 static inline int print_version(void) {
-        puts(GIT_VERSION);
+        /* Dracut relies on the version being a single integer */
+        puts(STRINGIFY(PROJECT_VERSION));
         return 0;
 }
-- 
2.19.1

